changelog/11830.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+agent: fix timestamp format in log messages from the templating engine
+```
```

Comment:
Might also be relevant to vault to add an entry for `fix command shell quoting issue [GH-1477, GH-1456, GH-1463]` from the 0.26.0 changlog?

Reply:
Good point, I'll get that added!